Feat/copy-relevant-community-files-from-seedcase-project-repo

signekb opened the pull request:
From what I can tell, the files that already exist in the seedcase-project repo that should be move to this repo are: 
- CODE_OF_CONDUCT.md
- CONTRIBUTING.md

Are there others that I might have missed? 

(GOVERNANCE.md also exists but is blank)

signekb commented:
@lwjohnst86  CONTRIBUTING links to guide-entries* and Decisions posts**. Should these be moved to this repository as well or should I make changes to the CONTRIBUTING file so it links to these posts in the seedcase-project repo?

*guide entries:  Documentation-Driven Development, How to add posts, and Reviewing through pull requests
**decision posts: Why Discord and Why Quarto

lwjohnst86 commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,146 @@
+# Code of Conduct
+
+::: callout-note
+This is a slightly modified version of the Contributor Covenant Code of
+Conduct (see [attribution](#attribution) below).
+:::
```
```suggestion
> This is a slightly modified version of the Contributor Covenant Code of
> Conduct (see [attribution](#attribution) below).
```

This callout-note block is a Quarto/HTML thing, so removing from here.

lwjohnst86 commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,54 @@
+---
```
This whole file could be simplified by referring people to read the Community contributing file on the website, since the guides and decisions are generated by Quarto.

signekb commented:
@lwjohnst86 I have changed the callout block and simplified the contributing file to only include a reference to the seedcase-project website instead.

lwjohnst86 commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,26 @@
+---
+title: Contributing Guidelines
+listing:
+  - id: guide-entries
+    contents: guide-entries
+    type: default
+    sort: "title"
+    categories: true
+    fields: 
+      - "title"
+      - "author"
+      - "description"
+      - "date"
+  - id: decisions
+    contents: decisions
+    type: default
+    sort: "title"
+    categories: true
+    fields: 
+      - "title"
+      - "author"
+      - "description"
+      - "date"
+---
```
```suggestion
# Contributing Guidelines
```

lwjohnst86 reviewed (approved):
Looks great!
